_includes/scripts.html

```diff
@@ -86,7 +86,7 @@
 
             search.attr('aria-expanded', true);
             var button = document.getElementById('cb1-button');
-            var cba = new ComboboxAutocomplete(document.getElementById('search'), button, document.getElementById('search-results'));
+            var cba = new ComboboxAutocomplete(document.getElementById('main-search-input'), button, document.getElementById('search-results'));
```

Comment:
@habbes Can you use the object `search` (from line 47) or does the object differ from what is returned by calling `getElementById`?

Reply:
I think it differs cause it's created with jQuery, I believe it's wrapper around the native dom element. There's probably an method to extract the underlying element.